tests/standardise/test_standardise.py

```diff
@@ -636,3 +636,67 @@ def test_standardise_sorting_false(caplog):
     log_messages = [record.message for record in caplog.records]
 
     assert "20220928.nc" in log_messages[-1]
+
+
+def test_standardise_footprints_satellite_raises_error():
+    """
+    Tests standardise footprint raises value error when site and obs_region values are not supplied.
+    """
+    datapath = get_footprint_datapath("GOSAT-BRAZIL-column_SOUTHAMERICA_201004_compressed.nc")
+
+    satellite = "GOSAT"
+    network = "GOSAT"
+    domain = "BRAZIL"
```

Comment:
Was this using the same data `test_standardise_footprint_satellite()`? If so the domain (and file name) should be updated to be consistent here as well
```suggestion
    domain = "TEST_SA"
```